src/crypto/index.ts

```diff
@@ -3221,14 +3221,15 @@ export class Crypto extends TypedEventEmitter<CryptoEvent, CryptoEventHandlerMap
         });
     }
 
-    public preprocessOneTimeKeyCounts(oneTimeKeysCounts: Map<string, number>): Promise<void> {
-        const currentCount = oneTimeKeysCounts.get("signed_curve25519") || 0;
-        this.updateOneTimeKeyCount(currentCount);
-        return Promise.resolve();
-    }
+    public processKeyCounts(oneTimeKeysCounts?: Record<string, number>, unusedFallbackKeys?: string[]): Promise<void> {
+        if (oneTimeKeysCounts) {
+            this.updateOneTimeKeyCount(oneTimeKeysCounts["signed_curve25519"]);
```

Comment:
previously, if there was no `signed_curve25519` property, we would pass 0 here. Now we will pass `undefined`. I don't think that's correct?